sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -0,0 +1,177 @@
+## API Report File for "@azure/notification-hubs"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+import { CommonClientOptions } from '@azure/core-client';
+import { OperationOptions } from '@azure/core-client';
+import { ServiceClient } from '@azure/core-client';
+
+// @public
+export interface ADMInstallation extends DeviceTokenInstallation {
+    platform: "adm";
+}
+
+// @public
+export interface ADMMessage extends JSONNotificationMessage {
```

Comment:
nit: we don't special case acronyms, so really it should be `AdmMessage extends JsonNotificationMessage`